improver/wind_calculations/wind_downscaling.py

```diff
@@ -446,11 +445,13 @@ def _calc_u_at_h(self, u_in, h_in, hhere, mask, dolog=False):
             hup = h_in[upidx].flatten()
             hlow = h_in[loidx].flatten()
         uup = u_in.take(
-            upidx.flatten() + np.arange(0, upidx.size * u_in.shape[2],
-                                        u_in.shape[2]))
+            upidx.flatten() +
+            np.arange(0, upidx.size * u_in.shape[2],  # pylint: disable=E1136
+                      u_in.shape[2]))  # pylint: disable=E1136
         ulow = u_in.take(
-            loidx.flatten() + np.arange(0, loidx.size * u_in.shape[2],
-                                        u_in.shape[2]))
+            loidx.flatten() +
+            np.arange(0, loidx.size * u_in.shape[2],  # pylint: disable=E1136
+                      u_in.shape[2]))   # pylint: disable=E1136
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'E1136' is used to disable 'unsubscriptable-object' message emission](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4915701)